locator/tablets.cc

```diff
@@ -888,6 +888,66 @@ std::optional<uint64_t> load_stats::get_tablet_size(host_id host, const range_ba
     return std::nullopt;
 }
 
+lw_shared_ptr<load_stats> load_stats::reconcile_tablets_resize(std::unordered_set<table_id> tables, const token_metadata& old_tm, const token_metadata& new_tm) const {
+    lw_shared_ptr<load_stats> reconciled_stats { make_lw_shared<load_stats>(*this) };
+    load_stats& new_stats = *reconciled_stats;
+
+    for (table_id table : tables) {
+        const auto& old_tmap = old_tm.tablets().get_tablet_map(table);
+        const auto& new_tmap = new_tm.tablets().get_tablet_map(table);
```

Comment:
```
new_tm.tablets().get_tablet_map(table);
```

This can throw if finalized table was dropped, let's guard against that and just drop the table form the stats.